numpy/core/defchararray.py

```diff
@@ -40,13 +40,6 @@
 
 
 _globalvar = 0
-if sys.version_info[0] >= 3:
-    _unicode = str
-    _bytes = bytes
-else:
-    _unicode = unicode
-    _bytes = str
-_len = len
```

Comment:
I think this might be deliberate - so that `np.char.len` is _not_ the builtin `len`

Reply:
Can you clarify.

My understanding is you'd have to call np.char._len and then you'd get the same result as calling len

Also `grepri '\b_len[^g]'` shows one result in faraway numpy/linalg/lapack_lite/clapack_scrub.py.

Reply:
My understanding was that this file declared a function `def len`, and this was to avoid a namespace collision with that function.

It seems that that never happened (https://github.com/numpy/numpy/commit/7deb4d5db4282e1e1c860dda268a21c6979e9725#diff-5da76d3b1619159cf50c444904e8b553R28), and that actually the function was called `str_len` in the end

So actually this is fine.